pertpy/tools/_metadata.py

```diff
@@ -0,0 +1,116 @@
+from pathlib import Path
+
+import anndata
+import pandas as pd
+import scanpy as sc
+from scanpy import settings
+
+from pertpy.data._dataloader import _download
+
+
+class MetaData:
+    """Utilities to fetch cell or perturbation metadata."""
+
+    def __init__(self):
+        """Download cell line metadata from DepMap"""
+        cell_line_file_name = "sample_info.csv"
+        cell_line_file_path = settings.cachedir.__str__() + "/" + cell_line_file_name
+        if not Path(cell_line_file_path).exists():
```

Comment:
Might be a good idea to add a print statement above this to state that the DepMap metadata file has not been found and will therefore be downloaded now.